src/stdio_mgr/stdio_mgr.py

```diff
@@ -28,13 +28,13 @@
 
 import sys
 from contextlib import contextmanager
-from io import StringIO, TextIOBase
+from io import BufferedReader, BytesIO, StringIO, TextIOBase, TextIOWrapper
```

Comment:
Probably can just remove the `StringIO` import, I don't think it's used after the switch to `TextIOWrapper` as the base class for `TeeStdIn`?

*(FYI, I'm trying the GH 'review' feature out for the first time; hopefully I'm not doing something goofy with it. :-P)*

Reply:
StringIO is still being used for the fake stdout and stderr, in this patch.

Those should also be replaced with similar voodoo, and probably before the next release, but I only had real code to test with stdin. (tho I did some research yesterday to find similar scenarios which need a real-ish stdout/stderr)

Reply:
Looks like stdout and stderr are addressed as part of #26. I'm ok with this -- will merge shortly.